shared/common-adapters/floating-picker.native.js

```diff
@@ -69,6 +56,9 @@ const styles = Styles.styleSheetCreate({
     left: 0,
     right: 0,
   },
+  fontFamilyOpenSans: {
```

Comment:
An actual change that affects the `set-explode-time` popup too. The font in the picker items will match the rest of the app.

Reply:
should just pull this from globalStyles.fontRegular